packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -96,8 +108,17 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
 
     instance.implementationAddress = biconomySmartAccountConfig.implementationAddress ?? BICONOMY_IMPLEMENTATION_ADDRESSES_BY_VERSION.V2_0_0;
 
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    instance.defaultValidationModule = biconomySmartAccountConfig.defaultValidationModule;
+    if (biconomySmartAccountConfig.defaultValidationModule) {
+      instance.defaultValidationModule = biconomySmartAccountConfig.defaultValidationModule;
+    } else if (biconomySmartAccountConfig.module) {
+      instance.defaultValidationModule = await instance.checkAndCreateModule(biconomySmartAccountConfig);
```

Comment:
why does this need entire biconomySmartAccountConfig?

we would just need signer. and as per current code module type (which is very limited to only ECDSA and MultiChainECDSA) 

maybe time to think about which all default validation modules can be there in future in next 3 months. and like signer what do they require.
like multiownedecda would need one or more signers.

Reply:
Check again.
Do we have multi owned ecdsa ?